libvast/src/system/spawn_index.cpp

```diff
@@ -41,6 +42,18 @@ spawn_index(node_actor::stateful_pointer<node_state> self,
     return caf::make_error(ec::lookup_error, "failed to find filesystem actor");
   const auto indexdir = args.dir / args.label;
   namespace sd = vast::defaults::system;
+  vast::index_config index_config;
+  const auto* settings = get_if(&args.inv.options, "vast.indexing");
```

Comment:
Is that correct?

Reply:
Do you mean the name of the config key? That's what we used in the design doc.

Reply:
The config key. I think it's really confusing to use `indexing`, when there's also indexers and the index currently. Using `index` as a key makes much more sense to me given that we want to split what currently is the index into partition builders and the scheduler.

Reply:
I don't necessarily disagree, but I'd like to have @mavam chime in before making any changes to the naming part since iirc we did already discuss this previously.

Reply:
I'm not opposed to going with `index`. In fact, it was my primary suggestion. The reason why we went with `indexing` was ironically to avoid a naming conflict with the old index concept.